core/src/sumeragi/main_loop.rs

```diff
@@ -367,26 +360,29 @@ impl Sumeragi {
     }
 }
 
-fn suggest_view_change(
+fn broadcast_view_change(
     sumeragi: &Sumeragi,
     view_change_proof_chain: &mut ProofChain,
     current_view_change_index: u64,
+    should_append: bool,
 ) {
```

Comment:
```suggestion
fn broadcast_view_change(
    sumeragi: &Sumeragi,
    view_change_proof_chain: &mut ProofChain,
) {
```

pull other stuff out of this function